crates/astria-conductor/src/celestia/mod.rs

```diff
@@ -507,8 +517,28 @@ impl FetchConvertVerifyAndReconstruct {
         .await
         .wrap_err("failed fetching blobs from Celestia")?;
 
+        {
+            // allow: histograms require f64; precision loss would be no problem
+            #![allow(clippy::cast_precision_loss)]
+            histogram!(
+                BLOBS_PER_CELESTIA_FETCH,
+                NAMESPACE_TYPE_LABEL => NAMESPACE_TYPE_METADATA,
+            )
+            .record(new_blobs.len_header_blobs() as f64);
+        }
+
+        {
+            // allow: histograms require f64; precision loss would be no problem
+            #![allow(clippy::cast_precision_loss)]
+            histogram!(
+                BLOBS_PER_CELESTIA_FETCH,
+                NAMESPACE_TYPE_LABEL => NAMESPACE_TYPE_ROLLUP_DATA,
+            )
+            .record(new_blobs.len_rollup_blobs() as f64);
+        }
```

Comment:
I think these either need to be two separate histograms entirely, or we should record `new_blobs.len_header_blobs() + new_blobs.len_rollup_blobs()` in this one.  I don't think Prometheus does anything with the labels, but it also doesn't hurt to add them in case we move away from Prometheus in the future I guess.

Reply:
After discussion with @SuperFluffy and looking at the code in `metrics` and `metrics-exporter-prometheus` seems like the labels do actually generate separate time series, so this should be fine.